test/specs/behaviors/testHelper.tsx

```diff
@@ -60,25 +67,39 @@ export class TestHelper {
   public findRegexAndAssingCorrespondingInfoToArray(behaviorMenuItems: any) {
     behaviorMenuItems.forEach(behavior => {
       behavior.variations.forEach(variant => {
-        variant.text.split('\n').forEach(singleLineText => {
-          this.iterateRegexDefinitions(singleLineText, variant.name)
-        })
+        if (!variant.specification && !variant.text) {
+          this.createMissingDescriptionTest(variant.name)
+        }
+        if (!variant.specification) {
+          this.verifySpecificationTag(variant.name)
+        } else {
+          variant.specification.split('\n').forEach(singleLineText => {
+            this.iterateRegexDefinitions(singleLineText, variant.name)
+          })
+        }
       })
     })
   }
 
   public iterateRegexDefinitions(singleLineText: string, behaviorName: string) {
```

Comment:
singleLineText -> singleSpecLine, this would better reflect semantics of this parameter